File: app/src/main/java/com/sumalogos/webster/sumalogos/MyApplication.java
package com.sumalogos.webster.sumalogos;

import android.app.Application;
import android.content.Context;

/**
 * Created by webster on 20/01/18.
 */

public class MyApplication extends Application {
    private static MyApplication instance;

    public static Context getContext() {
        return instance.getApplicationContext();
    }

    @Override
    public void onCreate() {
        super.onCreate();

        instance = this;
    }
}

File: app/src/main/java/com/sumalogos/webster/sumalogos/MyApplication.kt
package com.sumalogos.webster.sumalogos

/**
 * Created by webster on 01/01/18.
 */
import android.app.Application
import com.sumalogos.webster.sumalogos.util.AppDatabase

/**
 * Created by webster on 01/01/18.
 */
class MyApplication : Application() {
    private var appDatabase: AppDatabase? = null

    fun getAppDatabase(): AppDatabase {
        if (appDatabase == null) {
            appDatabase = AppDatabase.getAppDatabase(this)
        }

        return appDatabase as AppDatabase
    }
}